api/grpcserver/debug_service.go

```diff
@@ -149,6 +154,27 @@ func (d DebugService) ProposalsStream(_ *emptypb.Empty, stream pb.DebugService_P
 	}
 }
 
+func (d DebugService) ChangeLogLevel(ctx context.Context, req *pb.ChangeLogLevelRequest) (*emptypb.Empty, error) {
+	level, err := zap.ParseAtomicLevel(req.GetLevel())
+	if err != nil {
+		return nil, fmt.Errorf("parse level: %w", err)
+	}
+
+	if req.GetModule() == "*" {
+		d.log.SetLevel(&level)
```

Comment:
one thing to test would be that it actually overwrites all child loggerss, i am not confident that it will work. 
if that won't work it will be ok to iterate over whole map